data/421/296/621/421296621.geojson

```diff
@@ -34,35 +35,35 @@
         "gp:id":"12722603"
     },
     "wof:country":"FR",
-    "wof:geomhash":"579e36eda2bd0dbcd09a4aced1e6df8a",
+    "wof:geomhash":"93e6f5b0a784824c552d4b21a1559cf0",
     "wof:hierarchy":[
         {
             "continent_id":102191581,
             "country_id":85633147,
             "county_id":102067553,
             "empire_id":136253037,
-            "localadmin_id":404358937,
+            "localadmin_id":404358941,
```

Comment:
So in this case, we're saying 

https://spelunker.whosonfirst.org/id/421296621/

is now parented by localadmin of 

https://spelunker.whosonfirst.org/id/404358941/

This seems correct if we had a single one-to-one mapping of postalcode to localadmin, but it looks like the postalcode in this case is actually _larger_ than the localadmin, no? I'm fine with this edit, as we're adding a "more correct" hierarchy, but ideally this postalcode would have more than one parent/hierarchy. Similar to what we do with New York City:

https://spelunker.whosonfirst.org/id/85977539/

Reply:
Oh, I did not know it! Thanks for the information, I would probably do another pass to manage the multiple hierarchy! 

Thank you for the example